games-util/gamemode/gamemode-1.4.ebuild

```diff
@@ -0,0 +1,117 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MULTILIB_COMPAT=( abi_x86_{32,64} )
+
+inherit meson multilib-minimal ninja-utils user
+
+DESCRIPTION="Optimise Linux system performance on demand"
+HOMEPAGE="https://github.com/FeralInteractive/gamemode"
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="https://github.com/FeralInteractive/gamemode.git"
+	GAMEMODE_GIT_PTR="master"
+	inherit git-r3
+else
+	GAMEMODE_GIT_PTR="${PV}"
+	SRC_URI="https://github.com/FeralInteractive/gamemode/releases/download/${GAMEMODE_GIT_PTR}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="BSD"
+SLOT="0"
+IUSE=""
+
+RDEPEND="
+	>=sys-apps/systemd-236[${MULTILIB_USEDEP}]
+	sys-auth/polkit
+	sys-libs/libcap
+"
+DEPEND="${RDEPEND}"
+
+DOCS="CHANGELOG.md LICENSE.txt README.md example/gamemode.ini"
+
+pkg_pretend() {
```

Comment:
I just wonder why this is in pkg_pretend, no one will have time to read it before it starts compiling so doesnt matter if its in here or pkg_postinst for example. **If** you need to check whether that kernel option is enabled, there are linux-* eclasses to do so. But seems like its not the case, after reading the message.

Reply:
I was just replicating what a lot of other packages do when checking for recommended kernel options: They do that before merging starts. And I always concluded that's because it leaves the opportunity to cancel emerge now and adjust those things, or decide against using it at all then.

I'll move that to postinst if you still think it makes more sense.

Reply:
Sure sure, it's fine.